Sources/Commands/SwiftRunTool.swift

```diff
@@ -70,6 +74,10 @@ struct RunToolOptions: ParsableArguments {
     @Flag(name: .customLong("repl"), help: "Launch Swift REPL for the package")
     var shouldLaunchREPL: Bool = false
 
+    /// If the executable should be launched in LLDB.
+    @Flag(name: .customLong("debugger"), help: "Launch the executable in a debugger session")
+    var shouldLaunchDebugger: Bool = false
```

Comment:
should this as well as `shouldLaunchREPL` become an "run mode" enum, so we dont have to deal with the option of both of them passed in etc?

Reply:
makes sense, I made `RunMode` an `EnumerableFlag` here.